src/fping.c

```diff
@@ -285,7 +285,7 @@ struct timezone tz;
 /* switches */
 int generate_flag = 0; /* flag for IP list generation */
 int verbose_flag, quiet_flag, stats_flag, unreachable_flag, alive_flag;
-int elapsed_flag, version_flag, count_flag, loop_flag, netdata_flag;
+int elapsed_flag, version_flag, count_flag, loop_flag, netdata_flag, reachable_flag;
```

Comment:
Only one variable should be enough I think (i.e. the variable that contains the value of the reachable flag)

Reply:
Removed the reachable_flag variable (taking min_reachable>0 value as the flag as proposed).